tests/Tests/500__jupyterhub/test-jupyterhub-api-telemetry.robot

```diff
@@ -0,0 +1,78 @@
+*** Settings ***
+Resource    ../../Resources/ODS.robot
+Resource    ../../Resources/Page/ODH/JupyterHub/LaunchJupyterHub.robot
+Resource    ../../Resources/Page/ODH/JupyterHub/LoginJupyterHub.robot
+Resource    ../../Resources/Page/ODH/JupyterHub/JupyterHubSpawner.robot
+Resource    ../../Resources/Page/OCPDashboard/OCPMenu.robot
+Library     SeleniumLibrary
+
+*** Test Cases ***
+Verify Telemetry Data Is Accessible
+    [Tags]  Sanity
+    ...     ODS-347
+    Set Library Search Order  SeleniumLibrary
+    ${result} =   Is Usage Data Collection Enabled
+    Should Be True    ${result}
+    ${boolean} =    Checks SegmentKey Is Exposes In Jupyterhub API
+    Should Be True    ${boolean}
+    ${handle} =    Get Window Handles
+    Switch Window    ${handle}[0]
+    Sleep    2s    msg=Wait for to change tab
+    Navigate To Cluster Settings Page
+    Disable "Usage Data Collection"
+    Switch Window    ${handle}[1]
+    Reload Page
+    ${boolean} =    Check Is API Exposes SegmentKey
+    Should Not Be True    ${boolean}
+    [Teardown]    Teardown For Checks SegmentKey Is Exposes In Jupyterhub API TestCase    ${handle}
+
+*** Keywords ***
+Navigate To Cluster Settings Page
+    [Documentation]     Navigate to Cluste Settings page
+    Menu.Navigate To Page    Settings    Cluster settings
+    Wait Until Page Contains    Usage Data Collection
+
+Teardown For Checks SegmentKey Is Exposes In Jupyterhub API TestCase
+    [Documentation]     Enables ussage data collection and close the browser
+    [Arguments]     ${handle}
+    Switch Window    ${handle}[0]
+    Wait Until Page Contains    Usage Data Collection
+    Enable "Usage Data Collection"
+    ${result} =   Is Usage Data Collection Enabled
+    Should Be True    ${result}
+    Close Browser
+
+Checks SegmentKey Is Exposes In Jupyterhub API
+    [Documentation]     Loads JupyterHub API and checks is API exposes segmentkey
+    Load JupyterHub API Page
```

Comment:
for more clarity, I think this kw should go in the TC. Otherwise, when you do the Switch window, it's not clear where these windows come from

Reply:
in this way, you will not need anymore  `Checks SegmentKey Is Exposes In Jupyterhub API`, but `Check Is API Exposes SegmentKey` will be enough. So you spare also a repetition